src/OpenTelemetry.Exporter.Jaeger/JaegerExporter.cs

```diff
@@ -37,36 +36,23 @@ public JaegerExporter(JaegerExporterOptions options)
             this.JaegerAgentUdpBatcher = new JaegerUdpBatcher(options);
         }
 
-        internal IJaegerUdpBatcher JaegerAgentUdpBatcher { get; }
+        internal JaegerUdpBatcher JaegerAgentUdpBatcher { get; }
 
         public override async Task<ExportResult> ExportAsync(IEnumerable<Activity> activityBatch, CancellationToken cancellationToken)
         {
-            await this.exportLock.WaitAsync().ConfigureAwait(false);
```

Comment:
Removed this outer lock. Didn't seem to be adding any value. The spec says we should not call SpanExporter.Export concurrently. We're currently not in compliance with that rule, which I want to fix in my next PR. But there's a lock in Append so this one isn't needed anyway.